homeassistant/components/light/reproduce_state.py

```diff
@@ -44,14 +44,16 @@
     ATTR_TRANSITION,
 ]
 
+# This list needs to be ordered by the precision of the color models
 COLOR_GROUP = [
-    ATTR_COLOR_NAME,
-    ATTR_COLOR_TEMP,
-    ATTR_HS_COLOR,
-    ATTR_KELVIN,
-    ATTR_PROFILE,
     ATTR_RGB_COLOR,
```

Comment:
`hs_color` is the source of truth and used for generating both `xy` and `rgb` colors in the state machine as per https://github.com/home-assistant/home-assistant/blob/cd8c281cb6daae3140817b6c8b71fb36cf23b263/homeassistant/components/light/__init__.py#L484-L487

So `hs_color` should have the highest priority. With `color_temp` being 2nd.

Reply:
Done